### ISSUE TITLE
Find 404 error in the test

### ISSUE THREAD

**Issue by qman19791123** [noahbuscher/Macaw#61]
Find this line of code in  “ PHP - s ”  404  error occurs

Original code
```php
$uri = dirname($_SERVER['PHP_SELF']).'/'.$params[0]; //29
```

Modify code
```php
$uri = strpos($params[0], '/') === 0 ? $params[0] : '/' . $params[0]; //29
```

**Comment by daveismyname** [comment]
Agreed, I've opened a PR for this.

**Comment by noahbuscher** [comment]
Merged #62!

**Comment by daltonsutton** [comment]
Changing this back to the original code solved this issue of everything being a 404 error. Just wanted to give a heads up this may still be iffy.
